### DISCUSSION TITLE
necesito ayuda con git

### DISCUSSION THREAD

**Discussion by thbastian** [community/community#108848] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

![index html - curso de html y css - Visual Studio Code 17_2_2024 21_45_46](https://github.com/community/community/assets/113720598/213d36ac-af15-44a2-9f2d-d5a8770caec2)

tengo un problema con esto, alguien sabe como resolverlo?? como veran cuando hago "git init" me aparecen archivos de otros proyectos que no tienen nada que ver con lo que estaba haciendo incluso me aparece juegos

**Comment by Akash1134** [comment]
Hi @thbastian , thanks for participating!

We take our [Code of Conduct](https://github.com/community/community/blob/0ca37cf594e438fd4aec3c7f3736b95e6b0b3567/CODE_OF_CONDUCT.md) very seriously and want to help ensure that everyone has a good experience free of antagonism and harassment. Unfortunately, we don’t currently have moderators for languages other than English. Until that changes, we need to ask that everyone use English here in the GitHub Community when posting. We’ll be locking any posts in languages other than English for now, including this one.
